pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -1308,7 +1308,7 @@ func (ctrl *PersistentVolumeController) provisionClaimOperation(claimObj interfa
 	tags[CloudVolumeCreatedForVolumeNameTag] = pvName
 
 	options := vol.VolumeOptions{
-		PersistentVolumeReclaimPolicy: v1.PersistentVolumeReclaimDelete,
+		PersistentVolumeReclaimPolicy: storageClass.ReclaimPolicy,
```

Comment:
when the storage class reaches the pv controller, is it guranteed that the ReclaimPolicy is non empty and has been filled with defaults if not specified ?